src/libs/firebase/firestore/getCookiesLastUpdate.ts

```diff
@@ -0,0 +1,32 @@
+import { env } from 'libs/environment'
+import { firestoreRemoteStore } from 'libs/firebase/firestore/client'
+import { RemoteStoreCollections, RemoteStoreDocuments } from 'libs/firebase/firestore/types'
+import { captureMonitoringError } from 'libs/monitoring'
+
+export const getCookiesLastUpdate = (): Promise<
+  | void
+  | {
+      lastUpdated: Date
+      lastUpdateBuildVersion: number
+    }
```

Comment:
dans quel cas, ça peut retourner `void` ?

---

il est préférable de réutiliser les types existants

```suggestion
  | CookiesLastUpdate
```

Reply:
Dans le cas où il y a une erreur (et que `catch` est appelé). Est-ce que je retourne undefined dans le `catch` pour simplifier le typing ?

Reply:
je comprends mieux, merci

pas besoin de fusionner `void` et `undefined`